src/logic/safe/transactions/gas.ts

```diff
@@ -190,19 +187,15 @@ const calculateMinimumGasForTransaction = async (
   return 0
 }
 
-export const estimateSafeTxGas = async (
-  safe: GnosisSafe | undefined,
+export const estimateExecTransactionGas = async (
```

Comment:
So for me the previous name made more sense, as it described what we are actually calculating

Reply:
We discusses a lot the naming of those functions because they were confusing us a lot, I changed to that because we thought it will be better to understand, but it's ok for me both, what do you guys think? @fernandomg  @nicosampler  @dasanra

Reply:
with the new way we wanted to point out that the estimation is for a TX that will be handled by the safe. 
`estimateSafeTxGas` and `calculateGasOf` was kind of confusing for us, we had to read the code in order to understand the meaning. TBH I don't like either of both but I don't have a better proposal.-

Reply:
This will be improved on #1752